docs/paas-plus/dev/platform-as-a-service-plus.md

```diff
@@ -0,0 +1,161 @@
+
+### Assumptions and Exceptions
+We assume customers are already using Spryker PaaS Cloud services. When not applicable, customer needs to migrate to our Cloud service.
+
+
+
+## Step 2: Onboarding to Spryker CI
+Once you finalize step 1, you will be onboarded to Spryker CI and benefit from Spryker quality gate and automated upgrades.
+
+### What do you need to provide?
+In order to get you started, please provide the email of your Spryker Cloud Admin to your CSM. Your Spryker Cloud Admin user will be a Super User, which mean they will have RW rights on Spryker CI (e.g. ability to change repository; ability to invite new users to their workspace).
+Once we receive your user email, we will provision a Spryker CI workspace for you. You will then receive the invitation to your Spryker CI workspace via email.
+
+<img width="831" alt="image2" src="https://user-images.githubusercontent.com/83701393/176354827-ff1d3378-9a3b-4e15-a51c-c7bc6b56b5cf.png">
+
+
+## 5 steps to get started with Spryker CI
+
+**Step 1. Go to your $(workspacename) workspace**
+You receive an email with a one-time invitation token, but your invitation is always accessible by signing in on Buddy.works. Create your account & accept the invitation to your workspace.
+<img width="496" alt="image3" src="https://user-images.githubusercontent.com/83701393/176354843-956e46fe-4eb7-407b-b33a-dc6aa11eb38b.png">
+
+
+**Step 2. View projects inside your workspace**
+Your Spryker CI workspace contains the Spryker Upgrader Service project, as well as a project for each of your environments (e.g. development, production, etc).
+<img width="424" alt="Image4" src="https://user-images.githubusercontent.com/83701393/176354875-7333d656-f659-4aae-b650-803d7309b5c6.png">
+
+
+**Step 3. Invite new users***
+*Feature only available for Super Users.
+
+Add new users to your project and assign user permissions. Two roles are available: Super Users (RW) and Regular Users (RO & ability to run pipelines).
+
+
+**Step 4. Connect your repository***
+*Feature only available for Super Users.
+
+Connect your repository to a project, before running the pipelines. Open the“Code” section, click on the kebab menu and select "Switch repository or Git provider”. 
+
+![gif1](https://user-images.githubusercontent.com/83701393/176354903-a9d0a669-af3f-449d-bde7-cfc696e96569.gif)
+
+
+Select your git provider & add a new integration. Authorization is possible via OAuth or Token. After connecting, choose “Clone repository from scratch”.
+
+![gif2](https://user-images.githubusercontent.com/83701393/176354926-c486ddfc-25e2-4ae6-b32d-d0dad14d4aee.gif)
+
+
+
+**Step 5. Push & have fun!**
+Main pipeline in each project is triggered when you commit & push from your connected repository. More information on Pipelines is provided below.       
+<img width="446" alt="Image 5 6" src="https://user-images.githubusercontent.com/83701393/176354944-9d097014-9f1e-4ec7-9788-287f447fde4a.png">
```

Comment:
![Uploading Image 5&6.png…]()

Reply:
Reupload this one

Reply:
<img width="446" alt="Image 5 6" src="https://user-images.githubusercontent.com/83701393/177125436-55089d5f-cf48-4a20-aea5-0bc4da922957.png">

Reply:
done